src/Service.Tests/SqlTests/RestApiTests/Find/DwSqlFindApiTests.cs

```diff
@@ -551,7 +549,15 @@ public class DwSqlFindApiTests : FindApiTestBase
                 $"FROM {_integrationTableName} " +
                 $"ORDER BY [publisher_id] ASC, [id] ASC " +
                 $"FOR JSON PATH, INCLUDE_NULL_VALUES"
-            }
+            },
+            {
+                "FindManyStoredProcedureTest",
```

Comment:
Looks like some tests are failing for DW, since this method might be returning false: 

`RestService::IsHttpMethodAllowedForStoredProcedure`

You may need to modify the config file or check why this is returning false for StoredProcedure on DW.

Reply:
Fixed this. seemed to be a missing entity. I added that to config and ran tests locally and all passed.